# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/public/epplets-extra16/e-cpu2.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- E-CPU2v0.2/COMPILE.orig-rock-maint	2003-08-29 08:34:25.000000000 +0200
+++ E-CPU2v0.2/COMPILE	2003-08-29 09:02:11.000000000 +0200
@@ -24,8 +24,8 @@
 echo $CC $CFLAGS `imlib-config --cflags` -c $EPPLET.c -o $EPPLET.o
 $CC $CFLAGS `imlib-config --cflags` -c $EPPLET.c -o $EPPLET.o
 
-echo $CC $CFLAGS `imlib-config --cflags` -o $EPPLET.epplet $EPPLET.o `imlib-config --libs` -lepplet
-$CC $CFLAGS `imlib-config --cflags` -o $EPPLET.epplet $EPPLET.o  `imlib-config --libs` -lepplet
+echo $CC $CFLAGS `imlib-config --cflags` -o $EPPLET.epplet $EPPLET.o `imlib-config --libs` -L$E16/lib -lepplet -lGL
+$CC $CFLAGS `imlib-config --cflags` -o $EPPLET.epplet $EPPLET.o  `imlib-config --libs` -L$E16/lib -lepplet -lGL
 
 # Install E-NetFlame to the Enlightenment /bin dir
 echo install -m 755 ./$EPPLET.epplet $EBIN/$EPPLET.epplet
